sympy/physics/continuum_mechanics/beam.py

```diff
@@ -2234,3 +2234,79 @@ def deflection(self):
         the three axes.
         """
         return self._deflection
+
+    def plot_shear_force(self, dir="all", subs=None):
+        """
+        Returns a plot for Shear force along all three directions
+        present in the Beam object.
+
+        Parameters
+        ==========
+
+        dir : string (default : "all")
+            Direction along which shear force plot is required.
+            If no direction is specified, all plots are displayed.
+
+        subs : dictionary
+            Python dictionary containing Symbols as key and their
+            corresponding values.
+        """
+
+        shear_force = self.shear_force()
+
+        if subs is None:
+            subs = {}
+        subsx = subs
+        subsy = subs
+        subsz = subs
```

Comment:
Here subsx, subsy, and subsz all have the same value, which is the value of subs and these are not being modified anywhere in this function. Can't subs be used everywhere instead of creating three new variables?

Reply:
Thanks for reviewing. I used separate variables just to avoid confusion but I totally agree with you. I will fix it in the next commit.